src/commands/miscellaneous/stats.ts

```diff
@@ -82,7 +82,7 @@ const secsPerDay: number = 86400;
 const formatUptime = (uptime: number): string => {
     // Total seconds translated to hours, minutes, seconds
     const days: number = Math.floor(uptime / secsPerDay);
-    const hours: number = Math.floor(Math.floor(uptime % secsPerHour) / secsPerHour);
+    const hours: number = Math.floor(Math.floor(uptime % secsPerDay) / secsPerHour);
```

Comment:
This was the minor mistake that created the error where hours would be left out.